frontend/containers/dashboard/users/table/cells/actions/component.tsx

```diff
@@ -30,6 +33,17 @@ export const CellActions = ({ row }: CellActionsProps) => {
   const canResendInvitation =
     invitation === InvitationStatus.Expired || invitation === InvitationStatus.Waiting;
 
+  const handleResendInvitation = () => {
+    inviteUser.mutate(
+      { emails: [id] },
+      {
+        onSuccess: () => {
+          queryClient.invalidateQueries(Queries.AccountUsersList);
+        },
+      }
```

Comment:
Do we need to invalidate this query? It doesn't look like any information changes to warrant a re-fetch of the users list.